starter-code/src/functions-and-arrays.js

```diff
@@ -48,6 +90,20 @@ var wordsUnique = [
   'bring'
 ];
 
+function uniquifyArray (array){
+  var newArray = [];
+  for (i = 0 ; i < array.length ; i++){
+    var index = array.indexOf(array[i]);
+    if (index == i){
+      newArray.push(array[i]);
+    }
+  }
```

Comment:
Podéis también evaluar array.indexOf( ) directamente si queréis. Pero es correcto